### ISSUE TITLE
Need test class for the class - FindHttpBasicAuthFinder.

### ISSUE THREAD

**Issue by sudiptosarkar** [k0r0pt/Project-Tauro#11]
Yes, I know the class name needs refactor. We need a test class made for this class.
Tests should not make real calls. Instead, calls must be mocked with Mockito and/or PowerMockito.
Examples of the testing pattern can be found in the other three dependency repos mentioned in the README.md.

**Comment by cyterdan** [comment]
Hi ! 

I'd like to start working on this.

**Comment by sudiptosarkar** [comment]
@cyterdan Sure... Go ahead. Raise a Pull Request when you're done and put me on review... 🙂

**Comment by cyterdan** [comment]
Seems there is an infinite loop when google is not reachable, is that by design ?

**Comment by sudiptosarkar** [comment]
@cyterdan Yes. That's for resiliency. If the attacking system's Internet goes down, it will wait till it's back up... Also because my Internet connection sucked when I wrote it... :smiley:
